src/actions/ui-state-actions.d.ts

```diff
@@ -91,16 +91,31 @@ export function removeNotification(
 
 /** SET_EXPORT_IMAGE_SETTING */
 export type SetExportImageSettingUpdaterAction = {
-  payload: {ratio?: string; resolution?: string; legend?: string};
+  payload: {
+    ratio?: string;
+    resolution?: string;
+    legend?: string;
+    center?: boolean;
+    exporting?: boolean;
+    processing?: boolean;
+  };
 };
 export function setExportImageSetting(newSetting: {
```

Comment:
you can use `setExportImageSetting(newSetting: SetExportImageSettingUpdaterAction['payload'])` as a way to avoid duplicating code